### DISCUSSION TITLE
Add support for dotnet in addition to mono

### DISCUSSION THREAD

**Discussion by redbaron** [microsoft/vcpkg#25903] [Ideas]
****

**Is your feature request related to a problem? Please describe.**

Github Actions upcoming  ubuntu-22.04  image (currently in open beta) doesn't ship mono, which breaks nuget based caching.

**Proposed solution**
detect presence of `dotnet` and use it instead of mono.

**Describe alternatives you've considered**
Adding an alias from `dotnet` to `mono` seems to work, but it is a hack

**Comment by qc00** [comment]
Would expect more synergies between Microsoft projects than this silence...

Aliasing `dotnet` to `mono` (as some online posts suggest) does not work since `vcpkg-tools` seem to call mono natively.

**Reply by clee704** [reply]
I also want to see this happen as we're moving to CBL-Mariner from Ubuntu and Mariner doesn't ship mono.
